### PR TITLE
NET-91 (Latency) Metrics improvements

### PR DESCRIPTION
**Pull request by bearni95**

### CODE REVIEW

File: test/integration/latency.test.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+    beforeEach(async () => {
+        tracker = await startTracker({
+            host: '127.0.0.1',
+            port: trackerPort,
+            id: 'tracker'
+        })
+        // disable trackers formAndSendInstructions function
+        // eslint-disable-next-line no-underscore-dangle
+        tracker._formAndSendInstructions = () => {}
```
Comment (harbu) [inline test/integration/latency.test.js]: Is this disabling necessary? Can be probably removed.

File: test/integration/latency.test.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        node1 = await startNetworkNode({
+            host: '127.0.0.1',
+            port: port1,
+            id: 'node1',
+            trackers: [tracker.getAddress()]
+        })
+
+        node2 = await startNetworkNode({
```
Comment (harbu) [inline test/integration/latency.test.js]: `node2` not used in this test. Can be removed?

File: test/integration/latency.test.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
+const { StreamMessage, MessageID, MessageRef } = require('streamr-client-protocol').MessageLayer
+
+const { startNetworkNode, startTracker } = require('../../src/composition')
+
+describe('TestLatency', () => {
```
Comment (harbu) [inline test/integration/latency.test.js]: It is convention to use camel case when we are referring to a specific class. Here we don't have a class, so maybe a free form description is more appropriate, e.g. `latency metrics`.

File: test/integration/latency.test.js [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+    const trackerPort = 32907
+
+    let node1
+    const port1 = 33978
+
+    let node2
+    const port2 = 33979
+
+    beforeEach(async () => {
+        tracker = await startTracker({
```
Comment (harbu) [inline test/integration/latency.test.js:13]: You may not need `tracker` in this test actually with only one node.

File: test/integration/latency.test.js [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+        await node1.stop()
+        await tracker.stop()
+    })
+
+    it('TestLatency should fetch empty metrics', async () => {
+        const metrics = await node1.metrics.report()
+        expect(metrics.latency.last).toEqual(0)
+    })
+
+    it('Should send a single message to Node1 and collect latency', (done) => {
```
Comment (harbu) [inline test/integration/latency.test.js]: start description with lowercase to be consistent with other tests

File: test/integration/latency.test.js [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+                'msgChainId'
+            ),
+            prevMsgRef: new MessageRef(0, 0),
+            content: {
+                messageNo: 1
+            },
+        }))
+    })
+
+    it('Should send a bunch of messages to Node1 and collect latency', async (done) => {
```
Comment (harbu) [inline test/integration/latency.test.js]: start description with lowercase to be consistent with other tests

File: test/integration/latency.test.js [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+
+        node2.start()
+    })
+
+    afterEach(async () => {
+        await node1.stop()
+        await tracker.stop()
+    })
+
+    it('TestLatency should fetch empty metrics', async () => {
```
Comment (harbu) [inline test/integration/latency.test.js]: start description with lowercase to be consistent with other tests

**Review by harbu — CHANGES_REQUESTED** [review]

File: test/integration/latency.test.js (hunk 4 above)
Comment (bearni95) [reply test/integration/latency.test.js:13]: The node initialization throws an error when started without a tracker, seems to need it

**Review by harbu — APPROVED** [review]

**Pull request merged**